src/EditorFeatures/CSharpTest/Diagnostics/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -192,7 +192,7 @@ public void TestConflictingProperties()
         {
             Test(
 @"class Test : [|I1|] { int Prop { get ; set ; } } interface I1 { int Prop { get ; set ; } } ",
-@"using System ; class Test : I1 { int I1 . Prop { get { throw new NotImplementedException ( ) ; } set { throw new NotImplementedException ( ) ; } } int Prop { get ; set ; } } interface I1 { int Prop { get ; set ; } } ");
+@"class Test : I1 { int I1 . Prop { get ; set ; } int Prop { get ; set ; } } interface I1 { int Prop { get ; set ; } } ");
```

Comment:
@CyrusNajmabadi I agree.  So we should have an option for this. My initial assumption was that we should generate autoprops  when possible unless they are readonly or writeonly.  Do we think that is a good option and have the other option be to generate property bodies?